near-sdk-macros/src/lib.rs

```diff
@@ -470,6 +454,8 @@ fn process_impl_block(
 ///     fn sum(&self, a: u128, b: u128) -> u128;
 /// }
 /// ```
+/// 
+/// See more information about role of ext_contract in [NEAR documentation](https://docs.near.org/build/smart-contracts/anatomy/crosscontract)
```

Comment:
The documentation page may be removed or changed someday, so I recommend making this example complete as well as keeping a reference to the full documentation

The following code demonstrates how to call an external contract
```rs
#[near]
impl Contract {
    pub fn multiply_by_five(&mut self, number: u64) -> Promise {
        ext_calculator::ext(self.calculator_account.clone())
            .with_static_gas(Gas::from_tgas(5))
            .mult(number, 5)
    }
}
```